www/_posts/2016-03-21-tools-release.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+author:
+    name: Steve Gill
+    url: https://twitter.com/stevesgill
+title:  "Cordova 6.1.0 Released!"
+categories: news
+tags: release tools
+---
+New versions of cordova tools are now live!
+
+* [cordova-lib@6.1.0](https://www.npmjs.org/package/cordova-lib)
+* [cordova@6.1.0](https://www.npmjs.org/package/cordova)
+* [plugman@1.2.0](https://www.npmjs.org/package/plugman)
+* [cordova-js@4.1.4](https://www.npmjs.org/package/cordova-js)
+
+Release Highlights include:
+* [CB-10880](https://issues.apache.org/jira/browse/CB-10880) Removed plugin pinning. Instead we have implemented automatic plugin fetching based on engine requirements. You can read more about our new engine fetching prosal at [https://github.com/cordova/cordova-discuss/blob/master/proposals/plugin-version-fetching.md](https://github.com/cordova/cordova-discuss/blob/master/proposals/plugin-version-fetching.md). This implementation will grab the newest version of a plugin that your project supports when running `cordova plugin add`. 
```

Comment:
I'm actually going to update this to point to @riknoll blog post on the subject once he publishes it. https://github.com/apache/cordova-docs/pull/559